documentation/Versioning.md

```diff
@@ -0,0 +1,59 @@
+# Versioning
+
+This document describes how versions are labelled, when we bring out new versions of I-analyzer, and what you can expect when updating to a new version.
+
+If you're a developer, you can use this document to determine what the label should be for your new version. After that, walk through the checklist in [Making a release](./Making-a-release.md) to publish the new version.
+
+## Release labelling
+
+I-analyzer versions are based on [semantic versioning](https://semver.org/), so we distinguish between _major_, _minor_, and _patch_ releases. However, note that I-analyzer is primarily a user application and does not offer a public API, so it does _not_ conform to semantic versioning.
+
+Since I-analyzer is not centered on an API, it's not always obvious what is considered "breaking" change. The lists below gives a more detailed breakdown of what can be included in a patch, minor, or major release.
+
+A _patch_ release can include:
+
+- Fixes to bugs
+- Performance improvements
+- Changes to the backend API
```

Comment:
This does not belong in a patch release in my opinion. It's either backwards-compatible and minor or backwards-incompatible and major.